examples/runners/wgpu/src/lib.rs

```diff
@@ -56,10 +58,15 @@ pub enum RustGPUShader {
     Mouse,
 }
 
-fn shader_module(shader: RustGPUShader) -> wgpu::ShaderModuleDescriptor<'static> {
+fn maybe_watch(
+    shader: RustGPUShader,
+    force_no_watch: bool,
+) -> Receiver<wgpu::ShaderModuleDescriptor<'static>> {
+    // This bound needs to be 1, because when we don't watch, we
```

Comment:
Strange comment

Reply:
Oh yeah, I'll fix that. It's talking about e.g. line 98 of this file, where the item is added to the channel on the same thread.